src/Sulu/Bundle/AdminBundle/Resources/js/components/PublishIndicator/README.md

```diff
@@ -3,17 +3,23 @@ The `PublishIndicator` is a simple component, which can be used to show the draf
 The `published` state is represented by a green circle.
 
 ```
-<PublishIndicator published={true} />
+<div style={{width: '10px'}}>
```

Comment:
example looks strange why this 10px would avaid the div here.

Reply:
This is just to make sure that it pops out if the component breaks.